docs/markdown/snippets/test_using_custom_target_executable.md

```diff
@@ -0,0 +1,4 @@
+## Use `custom_target` as test executable
+
+[[test]] function now accepts [[@custom_tg]] and [[@custom_idx]] for the
```

Comment:
Missing "The" at the beginning of the sentence.

Second link should fail the "update website" CI.